www/routes/docs/[...slug].tsx

```diff
@@ -27,6 +27,7 @@ interface Data {
 
 interface Page extends TableOfContentsEntry {
   markdown: string;
+  frontMatterData: Record<string, string>;
```

Comment:
Can you rename to just `data`?

Also should really be `unknown` or `Record<string, unknown>` as yaml can have non string values. This is not too important tho.